doc/api/cli.md

```diff
@@ -595,6 +595,29 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_UNHANDLED_REJECTION=state`
+<!-- YAML
+added: REPLACEME
+-->
+
+Setting this environment variable allows fine grained control over the behavior
+of unhandled rejections. This may be set to either one of `SILENT`, `WARN`,
```

Comment:
Silent is nice although `NONE` might be more appropriate? 

 I'd ideally prefer it if `ERROR` was named `strict` but I'm also fine with `ERROR`

Reply:
I used `SILENT` and `ERROR` because they represent what happens. `NONE` is probably also fine but with `STRICT` it is not clear what it stands for. At least that's how it feels like to me.

Any other opinions?

Reply:
I'm fine with either - although `ERROR` doesn't do what I'd expect it to do (I'd like it to exit _immediately_ on any rejection getting to `SetPromiseRejectCallback` when not synchronously handled.

Reply:
Do you want that to be absolutely sure all catch handlers are attached sync? We could add another state for this. I have no strong feeling about either. I just thought it will likely be best for users in real applications because some modules might add a handler later on.

Reply:
> Do you want that to be absolutely sure all catch handlers are attached sync? 

Well, this might be me being greedy but yes - I'd like to be able to exit immediately whenever there is no `catch` handler attached synchronously (before other microticks have passed). 

I think that a lot of us in core feel this way but are afraid to recommend it because of the semantics - if we have a `none` option which makes debugging really hard we might as well have a `strict` mode which makes debugging really easy :D